### DISCUSSION TITLE
City traffic/NPC cars

### DISCUSSION THREAD

**Discussion by roo-ham** [voxeltycoon/issues#360] [General]
How about ai car? ai car can be obstacle for bus/truck operation in large city.

**Comment by katepolak** [comment]
Thank you for the suggestion, but for next time we would appreciate if you could describe it a bit more in depth. For now I at least updated the title as that is enough to describe the suggestion.

**Comment by roo-ham** [comment]
Thanks a lot.
well I'll put down more description for when you see again it.

1. AI car means city traffic, correctly. they spawn in house and they despawn in their destination and few times later, they respawn and going back to origin.
2. Traffic size should have affected by city size.
3. But if company gives enough passenger transportation service, citizens should generate little traffic.
4. So player have cover and support traffic service whenever city grow up enough.

Maybe, a system is also needed to check if the transportation network is complicated for prevent that player making short, many traffic line.

**Comment by katepolak** [comment]
Thank you for that, appreciate the quick update and more then clear enough description.
